highfive/configs/rust-lang/rust.json

```diff
@@ -24,13 +24,13 @@
         "library/panic_abort":       ["libs"],
         "library/panic_unwind":      ["libs"],
         "library/proc_macro":        ["@petrochenkov"],
-        "src/librustc":             ["compiler-team", "compiler-team-contributors"],
+        "compiler/rustc_middle":     ["compiler-team", "compiler-team-contributors"],
         "src/librustdoc":           ["rustdoc"],
         "library/std":               ["libs"],
         "library/term":              ["libs"],
         "library/test":              ["libs"],
         "src/llvm-project":         ["@cuviper"],
-        "src/rustc":                ["compiler-team", "compiler-team-contributors"],
+        "compiler/rustc":            ["compiler-team", "compiler-team-contributors"],
```

Comment:
Yeah, this feels weird... I would expect this to be just compiler/

Reply:
This used to be src/rustc as a prefix for `src/rustc_*` I suspect, so this is probably right.

The librustc prefix was previously needed but we now longer have lib prefixes so we can drop it I think.

Reply:
I'm pretty confused now... should I drop line 27 altogether, and just leave this line, then?

Reply:
That would be my expectation, yes.

Reply:
We can probably just say "compiler" without the rustc prefix as well.